scipy/signal/windows/windows.py

```diff
@@ -2017,7 +2017,7 @@ def _fftautocorr(x):
         'rect', 'rectangular'): (boxcar, False),
     ('chebwin', 'cheb'): (chebwin, True),
     ('cosine', 'halfcosine'): (cosine, False),
-    ('exponential', 'poisson'): (exponential, True),
+    ('exponential', 'poisson'): (exponential, False),
```

Comment:
`signal.window.exponential` does not need additional argument actually. (`M` is only needed:[scipy\.signal\.windows\.exponential](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.windows.exponential.html))